app_levels/en.json

```
@@ -3,11 +3,11 @@
     "applevel_1":  "Installable",
     "applevel_2":  "Installable in all situations",
     "applevel_3":  "Can be updated",
-    "applevel_4":  "Single Sign On support",
+    "applevel_4":  "Backup and restore support",
     "applevel_5":  "Clean",
-    "applevel_6":  "Backup and restore support",
+    "applevel_6":  "Available for the community",
```

Uh I don't think this makes so much sense for regular user ... I dunno what to do with this, but those descriptions are really short to explain what this is about. Here's a proposal, idk ...

```suggestion
    "applevel_6":  "Open to contributions from the community",
```